packages/vue/src/index.js

```diff
@@ -43,7 +46,7 @@ function errorHtmlLoader(req, res, next) {
   res.locals.html = `
         <div 
             data-serveside-component="${component}" 
-            data-serveside-id="se_embed_react_ssr_${req.id}"
+            data-serveside-id="se_embed_vue_ssr_${req.id}"
```

Comment:
Can we change all of the `se_embed_react_ssr_` in the project to something like `serveside_embed_`?

I realize that the other code is a relic

Reply:
or maybe just `serveside_`?

Reply:
I won't address this in this PR

Reply:
Sorry missed your comment